client/lib/core/utils/extensions.dart

```diff
@@ -10,6 +11,7 @@ import 'package:data_models/chat/emotion.dart';
 import 'package:data_models/community/community.dart';
 import 'package:data_models/community/membership.dart';
 import 'package:provider/provider.dart';
+import 'package:universal_html/js.dart';
```

Comment:
Is this necessary here? I don't see any HTML parsing happening, at least in new code